src/shogun/labels/BinaryLabels.cpp

```diff
@@ -148,3 +149,53 @@ CLabels* CBinaryLabels::shallow_subset_copy()
 
 	return shallow_copy_labels;
 }
+
+namespace shogun
+{
+	Some<CBinaryLabels> binary_from_binary(CBinaryLabels* orig)
```

Comment:
que?

Reply:
these are helper methods, could also move the contents directly into the switch below

Reply:
cpp only, nevermind